apps/api/src/routers/admin.py

```diff
@@ -131,6 +133,31 @@ async def release_decisions() -> None:
         )
 
 
+@router.post("/rsvp-reminder", dependencies=[Depends(require_role([Role.DIRECTOR]))])
+async def rsvp_reminder() -> None:
+    """Send email to applicants who have a status of ACCEPTED or WAIVER_SIGNED
+    reminding them to RSVP."""
+    not_yet_rsvpd = await mongodb_handler.retrieve(
+        Collection.USERS,
+        {"status": {"$in": [Decision.ACCEPTED, Status.WAIVER_SIGNED]}},
+        ["_id", "application_data.first_name"],
+    )
+    personalizations = [
+        ApplicationUpdatePersonalization(
+            email=_recover_email_from_uid(str(record["_id"])),
+            first_name=str(record["application_data.first_name"]),
+        )
+        for record in not_yet_rsvpd
+    ]
+    await sendgrid_handler.send_email(
```

Comment:
Suggestion: could we include a log message for how many people are getting reminder emails?